src/wbi/commands/remote.py

```diff
@@ -21,38 +20,61 @@ def add_args(parser):
         default="hello",
         help="Template to use for job script (default: hello)",
     )
+    parser.add_argument(
```

Comment:
Since this is a conceptually a boolean flag, let's use a boolean argument from argparse instead of str. I'm thinking `--cluster` for what you're calling `"compute"`. If not specified, runs on head node.

Reply:
Converted the flag into boolean.